py-polars/tests/unit/datatypes/test_temporal.py

```diff
@@ -2183,7 +2183,7 @@ def test_strptime_with_invalid_tz() -> None:
         ComputeError, match="unable to parse time zone: 'foo'"
     ), pytest.warns(
         FutureWarning,
-        match="In a future version of polars, non-area/location time zones",
+        match="time zones other than those in `zoneinfo.available_timezones",
```

Comment:
the error message wasn't quite right to begin with, but it looks like pytest doesn't validate it - have reported upstream https://github.com/pytest-dev/pytest/issues/11032